mobilenet/README.md

```diff
@@ -20,9 +20,9 @@ There are two main ways to get this model in your JavaScript project: via script
 
 ```html
 <!-- Load TensorFlow.js. This is required to use MobileNet. -->
-<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@0.15.3"> </script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tensorflow/1.0.1/tf.js"> </script>
```

Comment:
thanks so much for the PR Charlie!

would you mind reverting this to jsdeliver? we've been trying to use jsdelivr across the board since it gives us good statistics (and no need for the tf.js at the end, just replacing 0.15.3 with 1.0.1 will work for jsdeliver).

thank you!

Reply:
Sure, no problem, sorry! Doing it now!